pkg/cmd/build/build.go

```diff
@@ -271,6 +271,16 @@ func OptsFromBuildInfo(manifestName, svcName string, b *model.BuildInfo, o *type
 		}
 	}
 
+	if len(b.Secrets) > 0 {
+		if o.Secrets == nil {
+			o.Secrets = make([]string, 0)
+		}
+		// add to the build the secrets at the manifest
+		for id, src := range b.Secrets {
+			o.Secrets = append(o.Secrets, fmt.Sprintf("id=%s,src=%s", id, src))
```

Comment:
Not in the scope for this pr: did you find hard to add this secret with this particular formatting? Should we refactor the `Secrets` to be an array of `struct{id string, src string}`?

Reply:
`Secrets` is an option for the `okteto build` command, which is already implemented. This PR enables that this option can be added to the services declared inside the build section of the manifest as the issue describes